app/helpers/application_helper.rb

```diff
@@ -9,7 +9,7 @@ def react_md(text, props = {}, opts = {})
   end
 
   def markdown(source)
-    Kramdown::Document.new(source).to_html.html_safe
+    Kramdown::Document.new(source || '').to_html.html_safe
```

Comment:
I am honestly surprised that this one hasn't bit me yet, but I'm glad I caught it. Yay tests!